uber/tests/models/test_attendee.py

```diff
@@ -321,6 +321,13 @@ def yes_to_no_volunteering(self, unset_volunteering):
             assert unset_volunteering.called
 
 
```

Comment:
This was probably something dumb I was doing, but I couldn't get it to actually run this test when it was part of that class.  Probably just human error, I'll try and move it there in a bit.